usaspending_api/etl/subaward_etl.py

```diff
@@ -68,6 +65,7 @@ def load_subawards(submission_attributes, db_cursor):
         # a matching parent award id, piid, and submission attributes
         award = Award.objects.filter(transaction__submission=submission_attributes,
```

Comment:
I know this code is already in dev, but something I found helpful when doing various submission loader fixes is make a discrete method on the model (in this case, `Award`) for these types of matches. That made it a lot easier to unit test the edge cases with good old model mommy.